### ISSUE TITLE
Some thoughts on the Metadata area (and follow / view profile buttons)?

### ISSUE THREAD

**Issue by ponkey364** [Glimesh/glimesh_app#31]
Happy timezone,

Had some thoughts on the location of the follow & view profile buttons on the channel page and just wanted to share them for any comments before implementing:

Rather than trying to squeeze the buttons into the stream title, would it be a better idea to put them in the metadata area like the below?

![IMG_701493B4BCC4-1 3](https://user-images.githubusercontent.com/16962286/152209179-4c3cefb2-c809-44fe-ba77-07cd74e9e864.JPEG)

Additionally, I wonder if it'd be worth extending the metadata area to fill the whole chat area apart from the text box, then close it whenever the text box is tapped?

**Comment by clone1018** [comment]
> Rather than trying to squeeze the buttons into the stream title, would it be a better idea to put them in the metadata area like the below?

I like having some of the non-important buttons hidden for sure! I was thinking maybe clicking on profile pictures could be a universal "visit profile" type action? I suspect Follow might be important enough to have as an always-shown button, but with the potential of the Support button as well I think having them in the dropdown might be good! 



> Additionally, I wonder if it'd be worth extending the metadata area to fill the whole chat area apart from the text box, then close it whenever the text box is tapped?

Sounds good to me.

**Comment by ponkey364** [comment]
> I was thinking maybe clicking on profile pictures could be a universal "visit profile" type action?

I agree, wasn't sure about how easy / difficult that'd be to tap when I wrote the issue, but looking now, we shouldn't have any problems there

> I suspect Follow might be important enough to have as an always-shown button, but with the potential of the Support button as well I think having them in the dropdown might be good!

I think I'll have a play around and prototype a few different things and post them here
